backend/build_docs.py

```diff
@@ -360,49 +397,55 @@ def get_yosys_rtdocs() -> None:
         'data/markdown/manpages',
         'data/markdown/OR_docs',
         'data/markdown/ORFS_docs',
+        'data/markdown/OpenSTA_docs',
         'data/pdf',
-        'data/rtdocs',
+        'data/html',
     ]
     purge_folders(folder_paths=docs_paths)
 
     os.makedirs('data/markdown/manpages', exist_ok=True)
     os.makedirs('data/markdown/OR_docs', exist_ok=True)
     os.makedirs('data/markdown/OR_docs/installation', exist_ok=True)
     os.makedirs('data/markdown/OR_docs/tools', exist_ok=True)
+    os.makedirs('data/markdown/OR_docs/general', exist_ok=True)
     os.makedirs('data/markdown/ORFS_docs', exist_ok=True)
     os.makedirs('data/markdown/ORFS_docs/installation', exist_ok=True)
+    os.makedirs('data/markdown/ORFS_docs/general', exist_ok=True)
+    os.makedirs('data/markdown/OpenSTA_docs', exist_ok=True)
     os.makedirs('data/pdf/OpenSTA', exist_ok=True)
-    os.makedirs('data/rtdocs', exist_ok=True)
+    os.makedirs('data/pdf/OR_publications', exist_ok=True)
+    os.makedirs('data/html', exist_ok=True)
 
-    get_yosys_rtdocs()
+    get_or_publications()
+    get_or_website_html()
     get_opensta_docs()
+    get_yosys_docs_html()
 
     clone_repo(
-        url='https://github.com/The-OpenROAD-Project/OpenROAD.git',
+        url='https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts.git',
         commit_hash=os.getenv(
-            'OR_REPO_COMMIT', 'ffc5760f2df639cd184c40ceba253c7e02a006d5'
+            'ORFS_REPO_COMMIT', 'b94834df01cb58915bc0e8dabf85a314fbd8fb9e'
```

Comment:
why update?

Reply:
No specific reason, I updated them to newer commit hashes.

Reply:
I would propose to have the evaluation harness CI set up before we start changing commit hashes.